api/v1alpha1/generated.proto

```diff
@@ -636,6 +636,28 @@ message LocalObjectReference {
   optional string name = 3;
 }
 
+// RouteActionTargetObjectReference identifies a target object for a route
+// action within a known namespace.
+message RouteActionTargetObjectReference {
+  // Target is the object to forward matched requests to.
+  //
+  // Support: Core
+  optional LocalObjectReference target = 1;
+
+  // TargetPort specifies the destination port to use for a Target. TargetPort
+  // can be the port name or port number of the Target. If unspecified and Target
+  // is a Service object consisting of a single port definition, that port will
+  // be used. If unspecified and Target is a Service object consisting of multiple
+  // port definitions, the Service port that matches the request's destination
+  // port will be used. If a match can't be found, an error will be reflected in
```

Comment:
> that matches the request's destination port will be used

To clarify, does this mean that if the request was HTTP on port 80, then it will be sent to port 80 of the upstream service?

Reply:
Per yesterday's meeting, I'm removing some of the defaulting behavior. I'll ping you when I push a new commit to review.